main.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+import pandas as pd
+import json, os
+import spacy
+
+
+nlp = spacy.load('en_core_web_md')
+
+
+def load_emojis():
+    rows = []
+    with open('./emojis.json/emojis.json') as f:
+        for emoji in json.loads(f.read()):
+            rows.append([emoji['name'], emoji['unicode'], ' '.join(emoji['keywords']), emoji['definition']])
+    return np.array(rows)
+
+
+emojis = load_emojis()
+df = pd.DataFrame(emojis, columns=['name', 'unicode', 'keywords', 'definition'])
+print(df['unicode'].head())
+sentencevector = []
+count = 0
+df['definition'] = df['definition'].apply(lambda x: x.split(',')[0])
+for i in df['definition'] + df['name']:
```

Comment:
```python
for i in range(len(emojis)):
    desc = df['definition'][i] + df['name'][i]
    vecs = [token.vector for token in desc]
    mean_vec = np.average(vecs)
    sentencevector.append(mean_vec)
```

Reply:
Alright will do the changes and create a new pr